packages/css/src/modal/modal.css

```diff
@@ -40,6 +40,14 @@
   box-shadow: 8px 8px 16px var(--mdGreyColor60);
 }
 
+.md-modal__inner-wrapper--small {
+  width: 40vw;
```

Comment:
Gir det mening å innføre en min-size på modaler generelt for å forbedre dette problemet generelt, eller er problemstillingen forbeholdt small size?

Reply:
Jeg tenkte at det godt kan finnes caser hvor det er riktig at modalen tilpasser størrelse til innehold, slik det er i dag. Så jeg ville ikke ødelegge det noe, men heller utvide med mulighet for en fast størrelse med opt-in. 
Og hvis det viser seg at det er flere caser med slikt behov så går det jo ann å gjøre den funksjonaliteten enda mer sofistikert med flere størrelser, f.eks. small, medium, large. Eller rett og slett custom ved å spesifisere ønsket bredde/høyde som input eller noe slikt

Reply:
Nice. Da har du tenkt tanken i hvert fall. Jeg var btw på vei til å godkjenne i det du merget, men tenkte jeg bare skulle lufte tanken. Bra jobba. :)

Reply:
Snakket litt mer med Aurora om det nå, kommer en ny pr asap med forslag som gjør det mer customizable :)